_posts/databases/postgresql/timescaledb/2000-01-01-downsampling.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Downsampling
+nav: Downsampling
+modified_at: 2022-01-21 00:00:00
+tags: timescale databases postgresql extensions downsampling
+---
+
+Downsampling data allows to reduce the amount of data by simplifying the old values.
+
+## Example
+
+Bellow an example of a logic of downsampling. It is based on TimescaleDB documentation.
+
+The data is stored in a table `conditions`, composed by `temperature` and `humidity`
+value. This is a use case where a table is composed of raw datas.
+
+Now we want to decrease the database size. We don't need to have data by the
```

Comment:
nitpick: I'm not sure about the purpose of the word `Now` here. From my point of view it does not add any information.

```suggestion
We want to decrease the database size. We don't need to have data by the
```